cardinal/server/server_test.go

```diff
@@ -673,141 +673,150 @@ func TestTransactionReceiptReturnCorrectTickWindows(t *testing.T) {
 	tickCount = reply.EndTick - reply.StartTick - 1
 	assert.Equal(t, historySize-3, tickCount)
 }
-
-func TestCanGetTransactionReceipts(t *testing.T) {
-	// IncRequest in a transaction that increments the given number by 1.
-	type IncRequest struct {
-		Number int
-	}
-	type IncReply struct {
-		Number int
-	}
-
-	// DupeRequest is a transaction that appends a copy of the given string to itself.
-	type DupeRequest struct {
-		Str string
-	}
-	type DupeReply struct {
-		Str string
-	}
-
-	// ErrRequest is a transaction that will produce an error
-	type ErrRequest struct{}
-	type ErrReply struct{}
-
-	incTx := ecs.NewTransactionType[IncRequest, IncReply]("increment")
-	dupeTx := ecs.NewTransactionType[DupeRequest, DupeReply]("duplicate")
-	errTx := ecs.NewTransactionType[ErrRequest, ErrReply]("error")
-
-	world := inmem.NewECSWorldForTest(t)
-	assert.NilError(t, world.RegisterTransactions(incTx, dupeTx, errTx))
-	// System to handle incrementing numbers
-	world.AddSystem(func(world *ecs.World, queue *ecs.TransactionQueue, _ *ecs.Logger) error {
-		for _, tx := range incTx.In(queue) {
-			incTx.SetResult(world, tx.TxHash, IncReply{
-				Number: tx.Value.Number + 1,
-			})
+func TestCanGetTransactionReceiptsSwagger(t *testing.T) {
+	swaggerQueryReceiptsEndpoint := "query/receipts/list"
```

Comment:
I took the original TestCanGetTransactionReceipts test and put it in a loop to generate a test for the original route and the new route and new server.

Reply:
Look at where the server is instantiated with the swagger path parameter, and where the request is called on the endpoint... that's the main difference.